katka/views.py

```diff
@@ -95,6 +101,57 @@ class SCMPipelineRunViewSet(FilterViewMixin, AuditViewSet):
         'release': 'scmrelease',
     }
 
+    def perform_update(self, serializer):
+        status = serializer.initial_data.get('status', None)
```

Comment:
Shouldn't this be verified data?

Reply:
Good catch! First I had initial_data, but I changed it (not here) to validated_data). I'll update it!